app/assets/main/react_components/routes/lifestyle_footprints/contexts/SessionContext.js

```diff
@@ -0,0 +1,33 @@
+import React, { useContext } from 'react';
+
+const SessionContext = React.createContext(undefined);
+export const useSession = () => {
+  const context = useContext(SessionContext);
+  if (!context)
+    throw new Error(
+      'Please wrap component in SessionProvider to use useSession'
+    );
+  return context;
+};
+
+export const SessionProvider = ({
+  children,
+  currency,
+  lang,
+  currentRegion,
+  isUserSignedIn,
+}) => {
+  const sessionData = {
+    currentRegion: currentRegion.id,
+    slug: currentRegion.slug ? '/' + currentRegion.slug : '',
+    lang: lang,
+    currency: JSON.parse(currency),
+    signedInUser: isUserSignedIn,
```

Comment:
Är detta en bool eller en user? om det är en bool tycker jag den ska heta samma hela tiden, men det kanske finns en anledning till att den bytts?

Reply:
Det är en bool. Den har bytt namn för den gillar inte att man skickar in samma props-värde som man deklarerar. Den tycker props-värdet är unused då

Reply:
Misstänkte att det var något sånt!